### ISSUE TITLE
The results database should be moved to Cossan-X

### ISSUE THREAD

**Issue by FriesischScott** [cossan-working-group/OpenCossan#26]
In it's current state, we have possibility to save the analysis results in varying database formats. After most analyses you will find code along the lines of
```
if ~isdeployed
	... save results in database
end
```
However, there is no way to retrieve these results from the Matlab version. Only Cossan-X has this feature. I also think that no one using the Matlab version would be interested in saving the results in a database when the `.mat` files are already exported.

I think we should remove the complete database related code from the `development` version and have all necessary functionality in Cossan-X only.

Thoughts? @etheban @teobros

**Comment by etheban** [comment]
I disagree. We should have a full support of the database. It is not only a matter of storing .mat files but keeping track of the simulation.
It should be managed by the Analysis class

**Comment by FriesischScott** [comment]
Can you explain the use case here? Why would someone using Matlab want to save their results in a database where they are somehow less accessible than before?

Also what do you mean by "keeping track of the simulation"?

**Comment by etheban** [comment]
Because you can work in groups/team and storing results in a centralised location. You can also use the database for caching the results, eventually. 
Why a database should be less accessible than files with random names and no organisation?

**Comment by FriesischScott** [comment]
Ultimately, I think that everybody has their preferred way of storing results. It makes sense to export intermediate results somewhere so people can move them wherever they want, but mostly I think saving results should be left to the user the way they want it and are used to it.

If the results are in a database, you can't simply move them to wherever you want. You'd have to learn about databases first etc.
